rotkehlchen/start.py

```diff
@@ -0,0 +1,65 @@
+import sys
+import time
+from enum import StrEnum, auto
+from signal import SIGINT, SIGQUIT, SIGTERM, signal
+from subprocess import Popen  # noqa: S404
+from types import FrameType
+
+
+class ProcessType(StrEnum):
+    API_SERVER = auto()
+    DB_WRITER = auto()
+
+
+class ProcessData:
+    command: list[str]
+    process: Popen | None
+
+    def __init__(self, command: list[str], process: Popen | None) -> None:
+        self.command = command
+        self.process = process
+
+
+class ProcessesManager:
+    def __init__(self) -> None:
+        self.processes_running = True
+        self.processes: dict[ProcessType, ProcessData] = {
+            ProcessType.API_SERVER: ProcessData(
+                command=[sys.executable, '-m', 'rotkehlchen'] + sys.argv[1:],
```

Comment:
can you try calling pyinstaller manually and build only the backend? You can check package.py to see what logic is followed. I believe `sys.executable` doesn't exists there since what pyinstlaler does is bundle a python and then create an exe. Irc the frontend calls the binaries directly but check if this works